### DISCUSSION TITLE
Accordion item title styling

### DISCUSSION THREAD

**Discussion by AtharvaKatre** [facultyai/dash-bootstrap-components#980] [Q&A] [answered]
Currently the accordion item title only accepts string as a value for the title. 
Is it possible to pass a html children object to the title such that I can pass multiple div elements with their respective classnames for applying styling?
![image](https://github.com/facultyai/dash-bootstrap-components/assets/45846529/3cb73bd6-0213-44fa-b608-faaa0c9752a7)

**Comment by tcbegley** [comment]
Hi @AtharvaKatre 

Unfortunately I don't believe this is currently possible in Dash because of the way that `Accordion` works. See [this related issue](https://github.com/facultyai/dash-bootstrap-components/issues/769).

Here's a possible workaround though using CSS `::before` selector.

```python
import dash_bootstrap_components as dbc
from dash import Dash

app = Dash(external_stylesheets=[dbc.themes.BOOTSTRAP])

app.layout = dbc.Container(
    dbc.Accordion(
        [
            dbc.AccordionItem(
                f"The content of accordion item {i}",
                title=f"#{i}",
            )
            for i in range(1, 4)
        ]
    ),
    className="p-5",
)

if __name__ == "__main__":
    app.run_server(debug=True)
```

and some CSS in `assets/` folder

```css
.accordion-button::before {
  content: 'Collapsible group item';
  font-weight: 700;
  color: teal;
  padding-right: 5px;
}
```

Here's what it looks like for me

<img width="680" alt="image" src="https://github.com/facultyai/dash-bootstrap-components/assets/15220906/48f8dc89-b5d5-463d-90f5-a76d6fa95685">

Hope this helps.

**Comment by tcbegley** [comment, marked as answer]
This is now possible with the latest version of dash-bootstrap-components

```
pip install -U dash-bootstrap-components
```

```python
import dash_bootstrap_components as dbc
from dash import Dash, html

app = Dash(external_stylesheets=[dbc.themes.BOOTSTRAP])

app.layout = dbc.Container(
    dbc.Accordion(
        [
            dbc.AccordionItem(
                f"The content of accordion item {i}",
                title=[
                    html.Span(
                        "Collapsible group item",
                        style={"color": "blue", "fontWeight": 800},
                    ),
                    html.Span(f"#{i}", className="ms-1"),
                ],
            )
            for i in range(1, 4)
        ]
    ),
    className="p-5",
)

if __name__ == "__main__":
    app.run_server(debug=True)

```
